### DISCUSSION TITLE
Can't Dump Database Because Button Next Disabled

### DISCUSSION THREAD

**Discussion by chanurholis** [dbeaver/dbeaver#16716] [General]
#### System information: 
- Operating system (distribution) and version : Linux (Xubuntu 21.10)
- DBeaver version : 22.1.0
- Additional extensions : none

#### Connection specification:
- Database name and version : MySQL version 8
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
I can't dump database because button "Next >" is disabled
![image](https://user-images.githubusercontent.com/55186337/172171836-bad6fd45-6a6b-4468-b89a-e506f9150c34.png)

#### Steps to reproduce, if exist:
1. Select database
2. Right click
3. Tools
4. Dump database
5. Select table
6. Next
7. I found this error

**Comment by ShadelessFox** [comment]
Hello @chanurholis,

There's an error message describing what's wrong in the upper left corner.

You have to specify a local MySQL client to use the dump feature (it uses `mysqldump`).

**Comment by chanurholis** [comment]
> Hello @chanurholis,
> 
> There's an error message describing what's wrong in the upper left corner.
> 
> You have to specify a local MySQL client to use the dump feature (it uses `mysqldump`).

![image](https://user-images.githubusercontent.com/55186337/172174039-048ba942-2c2d-4a04-ae7b-4ff83458875c.png)
what should i fill in there?

**Comment by ShadelessFox** [comment]
You need to install the `mysql-client-core-8.0`:
```sh
sudo apt install mysql-client-core-8.0
```

Then click `Add Home` and select the directory containing the `mysqldump` executable.

It's often located under the `/usr/bin/` directory. Alternatively, you can use the following command to find it:

```sh
which mysqldump
```

**Comment by chanurholis** [comment]
> You need to install the `mysql-client-core-8.0`:
> 
> ```shell
> sudo apt install mysql-client-core-8.0
> ```
> 
> Then click `Add Home` and select the directory containing the `mysqldump` executable.
> 
> It's often located under the `/usr/bin/` directory. Alternatively, you can use the following command to find it:
> 
> ```shell
> which mysqldump
> ```

It's cool bro, solved. Thanks for response my issue 🆒

**Comment by maphew** [comment]
What other ways can the local client be installed for Windows? [In the wiki ](https://dbeaver.com/docs/dbeaver/Local-Client-Configuration/#local-client-configuration-for-windows) it says 

>In the Community Edition of DBeaver on Windows, local clients are downloaded from the internet when a remote client is selected in the Local Client options, typically during the first usage of database-specific tools such as backup, or restore functions.

..but that doesn't happen for me. The only option presented is to browse the file system for the client location.

![image](https://github.com/dbeaver/dbeaver/assets/486200/12960bcd-3bf5-41f2-8ee9-ff74fb3c7525)

**Reply by maphew** [reply]
The MariaDB server msi lets you choose to install only the client tools. After doing that using the DBeaver _Local client>>Browse_ to C: program files etc. worked.
